### ISSUE TITLE
Playback speed controls

### ISSUE THREAD

**Issue by omniraptorr** [KRTirtho/spotube#1526]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

want to make my music go fast!

### Suggested solution

Adding a selection menu to the player that lets you set playback speed

### Useful resources

Not a developer but this functionality seems to be built into android https://stackoverflow.com/a/37094353

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by Peperehobbit01** [comment]
The app is not just for Android, so saying it's built into Android does not realy matter here.

**Comment by s-b-repo** [comment]
true but we can prob still do it

**Comment by KRTirtho** [comment]
Yeah it's doable but who needs it though. Makes sense for a video player but not for an audio player.

**Comment by 4BlueTurtles** [comment]
> Yeah it's doable but who needs it though. Makes sense for a video player but not for an audio player.

I have a use case for this. I am a dancer, sometimes I wanna play music a little slower or faster to match the dance. As far as I am aware there are no free apps on Android which does this while being this well integrated with Spotify.

If you'd be willing to consider this I might be able to work on this feature.
